trojsten/contests/migrations/0011_auto_20180509_2154.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.12 on 2018-05-09 21:54
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contests', '0010_round_results_final'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
```

Comment:
Toto tu nemusi byt, nevyzera ze by sa menil tento field v modeli.

Reply:
migracia je automaticky generovana ... chces len aby sa to rucne zmazalo?